src/routes/settings/index.tsx

```diff
@@ -10,13 +10,15 @@ import NavBar from "~/components/NavBar";
 import { A } from "solid-start";
 import { For, Show } from "solid-js";
 import forward from "~/assets/icons/forward.svg";
+import { useI18n } from "~/i18n/context";
+import { DefaultTFuncReturn } from "i18next";
 
 function SettingsLinkList(props: {
     header: string;
     links: {
         href: string;
         text: string;
-        caption?: string;
+        caption?: string | DefaultTFuncReturn;
```

Comment:
looks like this type stuff can be solved with

https://www.i18next.com/overview/typescript#argument-of-type-defaulttfuncreturn-is-not-assignable-to-parameter-of-type-xyz

https://stackoverflow.com/questions/74143213/how-to-handle-i18next-new-type-defaulttfuncreturn